src/main/java/de/konfidas/ttc/validation/TimeStampValidator.java

```diff
@@ -0,0 +1,67 @@
+package de.konfidas.ttc.validation;
+
+import de.konfidas.ttc.exceptions.LogMessageValidationException;
+import de.konfidas.ttc.exceptions.ValidationException;
+import de.konfidas.ttc.messages.LogMessageImplementation;
+import de.konfidas.ttc.messages.LogMessage;
+import de.konfidas.ttc.messages.SystemLogMessage;
+import de.konfidas.ttc.messages.logtime.LogTime;
+import de.konfidas.ttc.tars.LogMessageArchive;
+import org.apache.commons.codec.binary.Hex;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.LinkedList;
+
+public class TimeStampValidator implements Validator{
+    final HashMap<String, LogTime> lastLogTime;
+
+    public TimeStampValidator(){
+        lastLogTime = new HashMap<>();
+    }
+
+    @Override
+    public Collection<ValidationException> validate(LogMessageArchive tar) {

```

Comment:
Hier verstehe ich erhlich gesagt die Prüfung nicht so ganz. Vielleicht können wir das auch kurz am Telefon besprechen

Reply:
Sure, 

again, for the interested audience: the log messages are considered sorted w.r.t to the signature counter. 
So all we have to check is, that the log message to be checked at the moment has a timestamp >=  `reference timestamp`, i.e. the timestamp of the last log message that we have seen. If this is the case, we have a not-smaller signature counter and a not-smaller timestamp, so the sequence of timestamp aligns with the sequence of signature counters. 

Missing at this validators is a consideration of _UpdateTime_ Events. In that case, when we check a Log Message, that is an _updateTime SystemLog_, we have to adopt the `reference timestamp` to the new set time. This has to be implemented, as soon as we are able to parse _updateTime SystemLogs_. 

What makes the code slightly confusing is the fact, that it supports multiple TSEs / TSE Serials. Therefore the reference timestamp to compare to is not one `timestamp`, but `HashMap<TSE_Serial, referenceTimestamp>`